### ISSUE TITLE
Remove PUTTY in favour of SSH with PowerShell?

### ISSUE THREAD

**Issue by Shrinks99** [tomeshnet/p2p-internet-workshop#74]
PUTTY is nice and all but Windows PowerShell includes SSH and works out of the box.  If we change this Windows users will follow the exact same steps as Mac & Linux users and won't have to install anything to access the Pis.

**Comment by Shrinks99** [comment]
I like the ability to have have as much parity between operating systems as possible for the instructions, the real question here is should we officially deprecate pre-Windows 10 machines here (and leave it up to the instructor to know what to do)?

I think it's decently fair to assume that if you're running Windows 10 it is updated to the point where it has SSH however according to this website 53% of people are running Windows 10 with 36% on Windows 7 globally.  The ciphers don't seem to be a problem for this scenario as Ryan was able to log in using PowerShell tonight without any problems.

**Comment by darkdrgn2k** [comment]
- Parity was initially proposed with git but struck down (granted it required additional installation)
https://github.com/tomeshnet/p2p-internet-workshop/pull/28#discussion_r179878766

- Windows "features" is a finicky beast and you cannot rely on them being installed especially on none-owned machines that do not have admin privileges.  Putty is the fastest way to get some one ssh on their computer without the need of "inner workings".  

- I would "wait and see" how ssh integration with windows develops over time.  IE dont mess with something that works when we have so little data.

Last i'm saying on the subject.  
Do with it what you will.

**Comment by Shrinks99** [comment]
Yeah based on the concerns presented by you(@darkdrgn2k) and @dcwalk in the chat I don't think it should be removed outright.  That said, I still think it could be added with PUTTY mentioned below to support older systems in order to streamline the in-class instructional process.

**Comment by benhylau** [comment]
I think it is reasonable to have PowerShell, then PUTTY instructions below it, because having to instruct someone to install PUTTY is an extra burden on the facilitators. For example, at conferences we are often running this with only 1 or 2 facilitators for a large group, having PowerShell would let most Windows users move on without help.

As mentioned in chat, we can probably move the SSH stuff into its own sheet.

**Comment by Shrinks99** [comment]
Along with SSH instructions this sheet also serves as a general jargon document with a section that allows participants to write in their own notes.
